libraries/SITL/SIM_XPlane.h

```diff
@@ -21,7 +21,7 @@
 #include <AP_HAL/AP_HAL_Boards.h>
 
 #ifndef HAL_SIM_XPLANE_ENABLED
-#define HAL_SIM_XPLANE_ENABLED (CONFIG_HAL_BOARD == HAL_BOARD_SITL)
+#define HAL_SIM_XPLANE_ENABLED 0
```

Comment:
umm, removing xplane?!?!?

Reply:
Just temporarily - this is a WIP and I wanted feedback before I did whatever needs doing to make xplane work again.

Reply:
XPlane is back